### DISCUSSION TITLE
How to get current ethereum price and convert useFeeData to usd unit?

### DISCUSSION THREAD

**Discussion by liho00** [wevm/wagmi#198] [Q&A] [answered]
I able to get useFeeData and its gasPrice, I have the idea of converting it to USD but I have no way to get the latest ETH price from wagmi.

Question: How do i get the latest eth price? (Does wagmi able to fetch the latest eth price?)

Could I suggest to add a usePrice hook for this library? If yes, please let me know the possible implementations, I would like to make a pr on this awesome library...

**Comment by tmm** [comment, marked as answer]
There's no way to get the current price using ethers.js (AFAIK). In the meantime, you could use the CoinGecko API for your project (free up to 50 requests/second).

https://www.coingecko.com/en/api/documentation
